subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/AvitoRemoteStorageTransport.kt

```diff
@@ -0,0 +1,26 @@
+package com.avito.android.test.report.transport
+
+import com.avito.android.test.report.ReportState
+import com.avito.android.test.report.model.TestMetadata
+import com.avito.filestorage.FutureValue
+import com.avito.filestorage.RemoteStorage
+
+internal class AvitoRemoteStorageTransport(
+    private val avitoFileStorage: RemoteStorage
+) : Transport {
+
+    override fun sendReport(state: ReportState.Initialized.Started) {
+        throw IllegalStateException("not implemented")
```

Comment:
Can we avoid this LSP violation?

Reply:
yes, after enabling toggle and removal of this class completely